modules/admission-webhook-create.adoc

```diff
@@ -0,0 +1,375 @@
+          secretName: server-serving-cert
+----
+<1> Note that the webhook server may expect a specific container name.
+<2> Points to a webhook server container image. Replace `<image_registry_username>/<image_path>:<tag>` with the appropriate value.
+<3> Specifies webhook container run commands here. Replace `<container_commands>` with the appropriate value.
+<4> Defines the target port within pods. This example uses port 8443.
+<5> Defines the target port within pods. This example uses port 8443.
+
+. Deploy the DaemonSet:
++
+----
+$ oc apply -f webhook-daemonset.yaml
+----
+
+. Define a secret for the service serving certificate signer, within a YAML file called `webhook-secret.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: my-webhook-namespace
+  name: server-serving-cert
+type: kubernetes.io/tls
+data:
+  tls.crt: <server_certificate>  <1>
+  tls.key: <server_key>  <2>
+----
+<1> References the signed webhook server certificate. Replace `<server_certificate>` with the appropriate certificate in base64 format.
+<2> References the signed webhook server key. Replace `<server_key>` with the appropriate key in base64 format.
+
+. Deploy the secret:
++
+----
+$ oc apply -f webhook-secret.yaml
+----
+
+. Define a service account and service, within a YAML file called `webhook-service.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: List
+items:
+
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    namespace: my-webhook-namespace
+    name: server
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    namespace: my-webhook-namespace
+    name: server
+    annotations:
+      service.alpha.openshift.io/serving-cert-secret-name: server-serving-cert
+  spec:
+    selector:
+      server: "true"
+    ports:
+    - port: 443  <1>
+      targetPort: 8443  <2>
+----
+<1> Defines the port the service listens on. This example uses port 443.
+<2> Defines the target port within pods, to which the service forwards connections. This example uses port 8443.
+
+. Expose the webhook server within the cluster:
++
+----
+$ oc apply -f webhook-service.yaml
+----
+
+. Define a Custom Resource Definition for the webhook server, in a file called `webhook-crd.yaml`:
++
+[source,yaml]
+----
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: namespacereservations.online.openshift.io  <1>
+spec:
+  group: online.openshift.io  <2>
+  version: v1alpha1  <3>
+  scope: Cluster  <4>
+  names:
+    plural: namespacereservations  <5>
+    singular: namespacereservation  <6>
+    kind: NamespaceReservation  <7>
+----
+<1> `<plural>.<group>` format and reflects Custom Resource Definition `spec` values. This example uses the `namespacereservations` resource.
+<2> REST API group name.
+<3> REST API version name.
+<4> Accepted values are `Namespaced` or `Cluster`.
+<5> Plural name to be included in URL.
+<6> Alias seen in `oc` output.
+<7> CamelCase reference for resource manifests.
```

Comment:
I'd remove "CamelCase" and just say "The reference for resource manifests". Technically this field is pascal case (a term I only recently learned) in that the first letter is capitalized (TestValue). Where as in camel case, the first letter is lowered (testValue).

Reply:
Thanks, that is useful to know for the future! I have now resolved this.